src/main/java/model/NumberGenerator.java

```diff
@@ -0,0 +1,5 @@
+package model;
+
+public interface NumberGenerator {
+	int generate();
+}
```

Comment:
NumberGenerator 인터페이스 👍 
구현체는 util 패키지로 분리하셨는데 이유가 있을까요? 보통 어떤 기준으로 util로 분리하시나요? 인터페이스로 나눴을 때 장점은 무엇이 있었을까요?

```suggestion

public interface NumberGenerator {
    int generate();
}
```

여기만 tab을 사용하고 계신데 보통 space 4칸을 사용합니다.

Reply:
원래는 `domain` - `infrastructure` 관계로, domain 패키지에 인터페이스를 두고 infrastructure 패키지에 구현체를 두는 편입니다. 패키지를 util에 그대로 둔건 실수입니다 🥲
인덴트는 반영했어요! 감사합니다~

Reply:
인터페이스로 나눴을 때 장점은.. 그 구현체를 바꾸더라도 원래 인터페이스에 의존하던 코드의 수정이 최소화 될 수 있다는 점, 테스트할 때 원하는대로 구현체의 동작을 설정할 수 있다는 점 등이 있겠습니다.